pvlib/tests/test_irradiance.py

```diff
@@ -28,7 +28,7 @@
 def times():
     # must include night values
     return pd.date_range(start='20140624', freq='6h', periods=4,
-                         tz='US/Arizona')
+                         tz='utc').tz_convert('US/Arizona')
```

Comment:
I had changed it to avoid failing of `test_get_extra_radiation_nrel_numba`. However, rather than doing that, I fixed the result of the failing test directly and put `times` back to the original in a08ca79 , thanks to your question :)